analyses/tmb-calculation/run_tmb_calculation.sh

```diff
@@ -12,35 +12,35 @@ script_directory="$(perl -e 'use File::Basename;
 cd "$script_directory" || exit
 
 # Set up paths to data files consumed by analysis
-data_path="../../data"
-scratch_path="../../scratch"
-bed_files_path="input"
+data_dir="../../data"
+scratch_dir="../../scratch"
+input_dir="input"
 
 # BED and GTF file paths
-cds_file="${scratch_path}/gencode.v27.primary_assembly.annotation.bed"
-wgs_bed="${scratch_path}/intersect_strelka_mutect2_vardict_WGS.bed"
+cds_file="${scratch_dir}/gencode.v27.primary_assembly.annotation.bed"
+wgs_bed="${scratch_dir}/intersect_strelka_mutect2_vardict_WGS.bed"
 
 # Filtered Fusion file 
-variant_file="${data_path}/snv-consensus-plus-hotspots.maf.tsv.gz"
+variant_file="${data_dir}/snv-consensus-plus-hotspots.maf.tsv.gz"
 
 # sample to BED mapping file
-mapping_file="${bed_files_path}/biospecimen_id_to_bed_map.txt"
+mapping_file="${input_dir}/biospecimen_id_to_bed_map.tsv"
 
 # Histology file
-histology_file="${data_path}/histologies.tsv"
+histology_file="${data_dir}/histologies.tsv"
```

Comment:
```suggestion
histology_file="${data_dir}/histologies-base.tsv"
```

I think we want to change this and then you can run it

Reply:
@jharenza, should I stop reporting the `cancer_group` associated with the sample in the results? The `histologies-base.tsv` file does not have `cancer_group` column.  That is the reason I did not regenerate the results (attached label `blocked` on the PR) waiting for the main `histologies.tsv` file to be released first. 
``` 
> read_tsv("snv-mutation-tmb-coding.tsv")
# A tibble: 2,764 x 6
   Tumor_Sample_Barcode experimental_st~ cancer_group mutation_count region_size
   <chr>                <chr>            <chr>                 <dbl>       <dbl>
 1 BS_CH0ZV29W          WGS              Neuroblasto~             21    35717401
 2 BS_4ZE1R1FW          WGS              Neuroblasto~             10    35717401
 3 BS_9YMT5HB4          WGS              Neuroblasto~              5    35717401
 4 BS_G6ASNYWP          WGS              Neuroblasto~             16    35717401
 5 BS_GPNWSW71          WGS              Neuroblasto~             77    35717401
 6 BS_20HSXMYQ          WGS              Neuroblasto~              6    35717401
 7 BS_PGBR1H2A          WGS              Neuroblasto~             18    35717401
 8 BS_KTHF096D          WGS              Neuroblasto~             12    35717401
 9 BS_HQYHVPC9          WGS              Neuroblasto~             11    35717401
10 BS_NWCPQC60          WGS              Neuroblasto~              2    35717401
# ... with 2,754 more rows, and 1 more variable: tmb <dbl>
```

Reply:
yeah, we don;t need it here anymore